README.md

```diff
@@ -1,10 +1,32 @@
 # automate
-Created with Electron JS and integrating with both Azure DevOps and GitHub, automate allows you to automate tasks such as changing branches across enterprise workspaces and solutions, tagging multiple repositories at the same time, and much more.
+automate is an Electron JS based source management program with Azure DevOps and Github integration. automate focuses on streamlining workflow tasks such as changing branches across enterprise workspaces and solutions, as well as tagging multiple repositories at the same time.
+
 
  [![Screenshot of the landing page.](images/landing.png)]()
 
-## Getting Started
-Make sure that you have [Node.js](https://nodejs.org/en/) installed. The minimum required version is 14.
+## Features
```

Comment:
Added list of features (current and future) establishes both a framework for streamlining the information a user wants to know about what automate does and what makes it stand out